client/src/main/java/io/pravega/client/stream/impl/PravegaCredsWrapper.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.stream.impl;
+
+import com.google.auth.Credentials;
+import java.io.IOException;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class PravegaCredsWrapper extends Credentials {
```

Comment:
I don't understand what this wrapper is for.   Why is the code using `MoreCallCredentials.from(...)`  at all?   Isn't that needlessly implicating `GoogleAuthLibraryCallCredentials `?  Why need an instance of `com.google.auth.Credentials`, as opposed to `io.grpc.CallCredentials`?

https://github.com/grpc/grpc-java/blob/master/auth/src/main/java/io/grpc/auth/MoreCallCredentials.java
https://github.com/grpc/grpc-java/blob/master/core/src/main/java/io/grpc/CallCredentials.java